.github/workflows/changelog.yml

```
@@ -0,0 +1,19 @@
+name: Check Changelog and Version
+
+on:
+  pull_request:
+    branches:
+      - master
+      - main
+
+jobs:
+  check-changelog:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 100
+      - uses: flotiq/check-changelog-and-version@v1.1.1
+        with:
+          changelog_file: CHANGELOG.md
+          version_file: package.json
```

```suggestion
          version_file: package.json
  check-changelog-plugin-manifest:
    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@v4
        with:
          fetch-depth: 100
      - uses: flotiq/check-changelog-and-version@v1.1.1
        with:
          changelog_file: CHANGELOG.md
          version_file: plugin-manifest.json
```
I forgot to put it in the ticket; plugins should check both pakage.json and plugin-manifest, as both of them should be updated